src/guides/upstream.md

```diff
@@ -0,0 +1,31 @@
+# Upstream to Another Proxy
+
+In order to send traffic to another proxy from Caido:
+
+1. Click on your account icon in the upper-rightmost corner of the Caido window.
+2. Select `Settings`.
+3. Select the `Network` tab or navigate to [http://localhost:8080/#/settings/network](http://localhost:8080/#/settings/network).
+
+::: warning LIMITATIONS
+
+- Currently, we do not support remote DNS resolution via SOCKS.
+- Calls to the [Caido Cloud](/concepts/internals/cloud.md) will not go through the additional proxy.
+:::
+
+4. Click the `Add Proxy` button in the appropriate panel depending on the type.
+5. Input the hostname or IP address and port number of the upstream proxy.
+6. Specify the scope of hosts that should be sent from Caido. To include all traffic, use `*`.
+7. You can test the configuration by clicking the `Test` button.
+
+9. Click the `+ Create` button to save the configuration.
+
+<img alt="Upstream HTTP proxy." src="/_images/http_proxy_new.png" center/>
+
+<img alt="Upstream SOCKS proxy." src="/_images/socks_proxy.png" center/>
+
+::: tip TIPS
+
+- Ensure the upstream HTTP proxy is running using a different port than Caido.
+- You can enable/disable usage using the toggle switch associated with the table row of your saved configuration.
+- Proxies can be edited or deleted with the pencil icon or trashbin buttons respectively.
+:::
```

Comment:
I would put an image of the table with the tips, you can also split it another h2 header